.ci/containerd_devmapper_setup.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
+
+if [ "$KATA_HYPERVISOR" != "firecracker" ]; then
+	echo "Devicemapper configuration is only for Firecracker"
+	exit
```

Comment:
Can you make this clearer (`echo "ERROR: ..."`, `echo "WARNING: ..."` or `echo "INFO: ..."`?) and also please specify an return code value to `exit`.

The same applies to other echo messages below.